Makefile

```
@@ -38,6 +38,13 @@ YQ_VERSION ?= v4.45.4
 # You can use podman or docker as a container engine. Notice that there are some options that might be only valid for one of them.
 ENGINE ?= docker
 
+# Version injection via ldflags
+GIT_COMMIT ?= $(shell git rev-parse --short HEAD 2>/dev/null || echo "unknown")
+BUILD_TIME ?= $(shell date -u '+%Y-%m-%dT%H:%M:%SZ')
+LDFLAGS = -X github.com/openshift-kni/lifecycle-agent/version.Version=$(VERSION) \
```

checkmake (0.3.2)

[warning] Variable "BUILD_TIME" possibly contains a timestamp and should be simply expanded.

(timestampexpanded)